geth/account/accounts.go

```diff
@@ -144,24 +144,34 @@ func (m *Manager) RecoverAccount(password, mnemonic string) (address, pubKey str
 	return address, pubKey, nil
 }
 
+type accountKeyFile struct {
+	Address string `json:"address"`
+}
```

Comment:
Can we make this struct inline? We're only using it in a single place for a single reason.